frontend/src/js/query-node-editor/QueryNodeEditor.tsx

```diff
@@ -158,10 +160,20 @@ const QueryNodeEditor = ({ node, ...props }: QueryNodeEditorPropsT) => {
       ? RIGHT_SIDE_WIDTH_COMPACT
       : RIGHT_SIDE_WIDTH);
 
-  const { autoLabel, autoLabelEnabled, setAutoLabelEnabled } = useAutoLabel({
-    node,
-    onUpdateLabel: props.onUpdateLabel,
-  });
+  // TEMPORARILY DISABLED UNTIL FEATURE IS COMPLETE
+  // const { autoLabel, autoLabelEnabled, setAutoLabelEnabled } = useAutoLabel({
+  //   node,
+  //   onUpdateLabel: props.onUpdateLabel,
+  // });
+  // const nodeLabel =
+  //   autoLabelEnabled && autoLabel
+  //     ? autoLabel
+  //     : nodeIsConceptQueryNode(node)
+  //     ? node.label
+  //     : node.label || node.id;
```

Comment:
I temporarily disabled the autoLabel feature cc @MarcoKorinth until it's feature-complete